features/distro-resources/src/main/resources/services/basicui.cfg

```diff
@@ -1,5 +1,5 @@
 # Defining the default sitemap to use
-org.eclipse.smarthome.basicui:defaultSitemap=demo
+defaultSitemap=demo
```

Comment:
Hm, so is this actually a bug in our 2.5.0 release then?

Reply:
Yes when opening Basic UI in the demo you'll see the list of sitemaps instead of going directly to the demo sitemap.